v4/weaviate/graphql/neartextbuilder.go

```diff
@@ -6,6 +6,68 @@ import (
 	"strings"
 )
 
+// fldMover is a type representing field names of a move sub query
+type fldMover string
+
+const (
+	fldMoverConcepts fldMover = "concepts"
+	fLDMoverForce    fldMover = "force"
```

Comment:
`fLDMoverForce` -> maybe fix to `fldMoverForce` to match naming convention of the other values

Reply:
Typo of course 😁 will be fixed